services/QuillLMS/spec/workers/user_requested_admin_verification_email_worker_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe UserRequestedAdminVerificationEmailWorker do
+  let(:job) { described_class.new}
+  let(:admin) { create(:admin) }
+
+  describe '#perform' do
+    it 'should send the stats email' do
```

Comment:
I suspect that this is a copy/paste thing, but it's probably worth tweaking this description to match the actual email we expect to be sent.